uniffi_bindgen/src/interface/mod.rs

```diff
@@ -241,6 +241,42 @@ impl<'ci> ComponentInterface {
         }
     }
 
+    /// Identify if we are using an unsigned type in kotlin
+    ///
+    /// In Kotlin, unsigned types are expiremental and will cause a warning if used without annotations
+    /// This should be removed when kotlin version
+    ///  > v1.5 https://kotlinlang.org/docs/whatsnew15.html#stable-unsigned-integer-types
+    pub fn contains_unsigned_type(&self, type_: &Type) -> bool {
```

Comment:
Another naming consistency nit: we have `type_contains_object_references` as plural with a leading `type_` prefix, and `contains_unsigned_type` as singular with no prefix. I think it's worth striving for consistency in the names of methods that do similarly-shaped things, especially given how much API surface this `ComponentInterface` struct exposes.

Reply:
I think i was trying to avoid a semi-weird name like `type_contains_unsigned_types` but I think I'll move to that for consistency